site/content/docs/1.20/config/request-routing.md

```diff
@@ -157,7 +157,7 @@ Building on multiple upstreams is the ability to define relative weights for ups
 This is commonly used for canary testing of new versions of an application when you want to send a small fraction of traffic to a specific Service.
 
 ```yaml
-# httpproxy-weight-shfiting.yaml
+# httpproxy-weight-shifting.yaml
```

Comment:
The file is renamed above...

This is potentially problematic, as these are in theory docs for a different version where the file wouldn't have been renamed. It depends on how people are expecting to read this item...